<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>walletDialog</class>
 <widget class="QDialog" name="walletDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wallet</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>301</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Payment Wallet</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>284</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbBank">
       <property name="text">
        <string>Bank:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbbBank"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbAccount">
       <property name="text">
        <string>Account:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leAccount"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbToken">
       <property name="text">
        <string>Token:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="leToken">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnQuery">
         <property name="text">
          <string>Query</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbBalance">
       <property name="text">
        <string>Balance:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="leBalance">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbUnit">
         <property name="text">
          <string>(mBTC)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCharge">
         <property name="text">
          <string>Charge...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>301</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Reserve Wallet</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="lbBlkChain">
       <property name="text">
        <string>Chain:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbbBlockChain"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbPassphrase">
       <property name="text">
        <string>Phrase:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="lePassphrase">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBlkChainQuery">
         <property name="text">
          <string>Query</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbBlkChainBalance">
       <property name="text">
        <string>Balance:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="leBlkChainBalance"/>
       </item>
       <item>
        <widget class="QLabel" name="lbBlkUnit">
         <property name="text">
          <string>(mBTC)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnImport">
         <property name="text">
          <string> Import...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>281</width>
      <height>113</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbDetails">
       <property name="text">
        <string>Details:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="lstDetails"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cbbBank</tabstop>
  <tabstop>leAccount</tabstop>
  <tabstop>leToken</tabstop>
  <tabstop>btnQuery</tabstop>
  <tabstop>leBalance</tabstop>
  <tabstop>btnCharge</tabstop>
  <tabstop>cbbBlockChain</tabstop>
  <tabstop>lePassphrase</tabstop>
  <tabstop>btnBlkChainQuery</tabstop>
  <tabstop>leBlkChainBalance</tabstop>
  <tabstop>btnImport</tabstop>
  <tabstop>lstDetails</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>walletDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>walletDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
